samples/apps/github-qna-webapp-react/src/components/setup/ModelConfig.tsx

```diff
@@ -98,12 +102,15 @@ const ModelConfig: FC<IData> = ({
             if (fetchModels) {
                 setIsBusy(true);
                 setErrorMessage(undefined);
+                console.log(`Fetching models: key ${resourceInput.key}, endpoint ${resourceInput.endpoint}`);
```

Comment:
Does this log the access key? If so, we should not log the key, or at the very least we should obfuscate (e.g. "sk-abc*********)

Reply:
Removed this log all together -- decided we won't really need it